mcp/client_test.go

```diff
@@ -202,9 +203,8 @@ func TestClientCapabilities(t *testing.T) {
 			name:            "With initial capabilities",
 			configureClient: func(s *Client) {},
 			wantCapabilities: &ClientCapabilities{
-				Roots: struct {
-					ListChanged bool "json:\"listChanged,omitempty\""
-				}{ListChanged: true},
+				Roots:   canListRoots,
```

Comment:
nit: there is mixed use of `Roots: canListRoots` (here) and in-lining its definition (line 254 below). imo, inlining it seems simpler, since the type name isn't too long, instead of introducing another variable for the reader to remember